app/api/resources/user.py

```diff
@@ -42,7 +42,6 @@ class UserList(Resource):
         responses={
             HTTPStatus.UNAUTHORIZED: f"{messages.TOKEN_HAS_EXPIRED['message']}<br>"
             f"{messages.TOKEN_IS_INVALID['message']}<br>"
-            f"{messages.AUTHORISATION_TOKEN_IS_MISSING['message']}"
         }
```

Comment:
Do not remove this message, instead of check why this error is not pop up.
Because per the description of the endpoint 'A user with valid access token can use this endpoint to edit his/her own
user details'